crates/sovereign-sdk/full-node/db/sov-db/src/schema/tables.rs

```diff
@@ -53,16 +53,15 @@ pub const STATE_TABLES: &[&str] = &[
 pub const LEDGER_TABLES: &[&str] = &[
     SlotByNumber::table_name(),
```

Comment:
we don't use slotbynumber by hash do tx by hash by number, event by ...

Reply:
did we not remove these because it was breaking too many tests?

Reply:
We use them only for setting next item numbers

Reply:
Talked with @eyusufatik and decided removing tables should be a separate pr